WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivityVideoHelper.java

```diff
@@ -155,6 +227,9 @@ public void onError(Exception exception) {
             if (isStopped) {
                 return;
             }
+
+            trackVideoProcessingEvents(true, exception);
```

Comment:
Because of the above check, we're not tracking the exception when `isStopped` is true. Is that on purpose?